source/index.md

```diff
@@ -20,7 +27,7 @@ To get started, you can:
 * [Build a new tax and benefit system](coding-the-legislation/bootstrapping_a_new_country_package.md) if it doesn’t exist already.
 * [Contribute](contribute/index.md) to an existing system by adding or improving elements of the legislation.
 
-Then, you will turn legal code into OpenFisca code, which is a subset of the Python programming language with many helpful tools:
+With many helpful OpenFisca tools, you can turn legal code into OpenFisca code which is a subset of the Python programming language:
```

Comment:
Here, the suggested change does not work: the intention is really to say that OpenFisca is “a subset of the Python programming language with many helpful tools”. It is not about “turning legal code into OpenFisca code with many helpful tools”. Do you have another suggestion to make that sentence clearer? 🙂

Reply:
Then it would be more approriate to add the line " OpenFisca is a subset of the Python programming language with many helpful tools” at the beginning of the doc.
And replace the above  sentence with maybe: " Next, you will turn the legal code to OpenFisca code by following the below steps" or something along that logic?

Reply:
I see your point, thanks for this suggestion!

I don't think we can say that OpenFisca _is_ a subset of Python. The OpenFisca [_DSL_](https://en.wikipedia.org/wiki/Domain-specific_language) is “a subset of the Python programming language with many helpful tools”, but OpenFisca is more than its DSL.

This is really not clear indeed 😄 Thank you so much for identifying this rough edge! What would you think of the following:

```suggestion
Then, you will make legal code executable by writing it in the OpenFisca [DSL](https://en.wikipedia.org/wiki/Domain-specific_language), which is a subset of the Python programming language with dedicated functions and tools specifically targeted at modelling rules:
```